### DISCUSSION TITLE
如果开了只代理常用端口，因为没代理853端口造成DoT协议域名解析失败

### DISCUSSION THREAD

**Discussion by lvxj11** [nikkinikki-org/OpenWrt-nikki#267] [General]
最近想搞mihomo+mosdns，在mosdns日志有大量警报，排查是访问DoT的853端口失败。自定义常用端口添加853端口后警报消除。

**Comment by morytyann** [comment]
嗯，目前TCP常用端口没有853，国外的DoT服务确实会无法访问，我添加一下
